tests/unit/IndependentReserve/CurrencyTest.php

```diff
@@ -6,39 +6,108 @@
 
 class CurrencyTest extends TestCase
 {
-    public function testXbt()
+    public function testAud()
     {
-        $this->assert(Currency::XBT, equals, 'Xbt');
+        $this->assert(Currency::AUD, equals, 'Aud');
     }
 
     public function testUsd()
     {
         $this->assert(Currency::USD, equals, 'Usd');
     }
 
-    public function testAud()
+    public function testNzd()
     {
-        $this->assert(Currency::AUD, equals, 'Aud');
+        $this->assert(Currency::NZD, equals, 'Nzd');
     }
 
-    public function testNzd()
+    public function testXbt()
     {
-        $this->assert(Currency::NZD, equals, 'Nzd');
+        $this->assert(Currency::XBT, equals, 'Xbt');
     }
 
     public function testEth()
     {
         $this->assert(Currency::ETH, equals, 'Eth');
     }
 
+    public function testXrp()
+    {
+        $this->assert(Currency::XRP, equals, 'Xrp');
+    }
+
+    public function testXrp()
+    {
+        $this->assert(Currency::XRP, equals, 'Xrp');
+    }
+
     public function testBch()
     {
         $this->assert(Currency::BCH, equals, 'Bch');
     }
 
+    public function testBsv()
+    {
+        $this->assert(Currency::BSV, equals, 'Bsv');
+    }
+
+    public function testUsdt()
+    {
+        $this->assert(Currency::USDT, equals, 'Usdt');
+    }
+
+    public function testUsdt()
+    {
+        $this->assert(Currency::USDT, equals, 'Usdt');
+    }
```

Comment:
There are two definitions of `testUsdt`.

Reply:
My apologies - late night copy and paste error...